the-event-loop/interval.js

```diff
@@ -0,0 +1,18 @@
+var counter = 3;
+function countdown() {
+  if (counter === 3) {
+    console.log('3');
+    counter--;
+  } else if (counter === 2) {
+    console.log('2');
+    counter--;
+  } else if (counter === 1) {
+    console.log('1');
+    counter--;
+  } else if (counter === 0) {
```

Comment:
Since the `counter` is the same as the number you want to log, you can just use that for the console log. For example:

```JAVASCRIPT
if(counter > 0) {
  console.log(counter);
  counter--;
} else {
  console.log('Blast Off');
  clearInterval(stop);  
}
```